piker/ui/_ohlc.py

```diff
@@ -146,7 +146,7 @@ def path_arrays_from_ohlc(
         # specifies that the first edge is never connected to the
         # prior bars last edge thus providing a small "gap"/"space"
         # between bars determined by ``bar_gap``.
-        c[istart:istop] = (0, 1, 1, 1, 1, 1)
+        c[istart:istop] = (1, 1, 1, 1, 1, 0)
```

Comment:
Hmm i feel like this got reverted since they fixed the bug in `pyqtgraph` that caused me to change this in the first place?

Reply:
Actually on latest `pg` master we seem to be good.
So I think all the issues I noticed are good now?

Reply:
Added a dev branch pin back to `requirements.txt`; prolly a better approach for now anyway..